src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -670,173 +637,26 @@ export const handleEncryptedSubmission = [
   CaptchaMiddleware.validateCaptchaParams,
   TurnstileMiddleware.validateTurnstileParams,
   EncryptSubmissionMiddleware.validateEncryptSubmissionParams,
+  EncryptSubmissionMiddleware.createFormsgReqBody,
+  EncryptSubmissionMiddleware.retrieveForm,
+  EncryptSubmissionMiddleware.checkPublicKey,
+  EncryptSubmissionMiddleware.moveEncryptedPayload,
+  EncryptSubmissionMiddleware.checkEncryptMode,
```

Comment:
proposed refactor: Suggest to group this as 'processEncryptedSubmission', i.e.

const processEncryptedSubmission = [EncryptSubmissionMiddleware.createFormsgReqBody, <...>]
and then export const handleEncryptedSubmission = [<...>, ...processEncryptedSubmission]

Reply:
may want to group in EncryptSubmissionMiddleware as an export, so its cleanly separated from the controller logic

Reply:
As discussed in our huddle, addressed here: [e081264](https://github.com/opengovsg/FormSG/pull/6595/commits/e081264c64aa0ebb92069bba5f1a7e8c5461c91d)
Slightly different naming for the combined middleware to highlight (imo) the more important parts of what it is doing though, which is creating the `formsg` field in the request body and retrieving the form definition.
Will close subsequent related comments so we can track all discussions here.